l1-contracts/test/foundry/l1/unit/concrete/AddressAliasHelper/actualRefundRecipient.t.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {AddressAliasHelperSharedTest} from "./_AddressAliasHelper_Shared.t.sol";
+import {Test} from "forge-std/Test.sol";
+
+// Used to test when recipient is a contract
+contract TestContract {
+    // add this to be excluded from coverage report
+    function test() internal virtual {}
+}
+
+contract actualRefundRecipient is AddressAliasHelperSharedTest {
```

Comment:
contract should be named from the capital letter

Reply:
Yes. Changed.